third_party/2and3/retry/api.pyi

```diff
@@ -0,0 +1,28 @@
+from logging import Logger
+from typing import Any, Callable, Dict, Optional, Sequence, Type, TypeVar, Union
+
+_T = TypeVar("_T", bound=Callable[..., Any])
+_Decorator = Callable[[_T], _T]
+_R = TypeVar("_R")
+
+def retry_call(
+    f: Callable[..., _R],
+    fargs: Optional[Sequence[Any]] = ...,
+    fkwargs: Optional[Dict[str, Any]] = ...,
+    exceptions: Union[Type[Exception], Sequence[Type[Exception]]] = ...,
```

Comment:
This should be `Tuple[Type[Exception], ...]`, not Sequence. The implementation uses it directly in an except clause (https://github.com/invl/retry/blob/master/retry/api.py#L34).

Reply:
Good catch. I just pushed a fix.